docs/content/latest/yugabyte-cloud/release-notes.md

```diff
@@ -15,6 +15,17 @@ showAsideToc: true
 
 ## Releases
 
+### November 18, 2021
+
+#### New features
+
+- Monitor additional performance metrics for your cluster to ensure the cluster configuration matches its performance requirements using the cluster **Performance Metrics** tab. New metrics include YSQL and YCQL operations per second, YSQL and YCQL latency, network bytes per second, and more.
```

Comment:
I think it's always better when list items consistently start with a noun or gerund in release notes. E.g: 
Additional performance metrics for a cluster ...